cron/worker/worker_test.go

```diff
@@ -55,3 +55,49 @@ func TestResultFilename(t *testing.T) {
 		})
 	}
 }
+
+func asptr(s string) *string {
+	return &s
+}
+
+func TestGitlabRepo(t *testing.T) {
```

Comment:
what is this test trying to do?
it seems like a copy/paste of the test above it with a minimal change that doesnt affect output?